share/gpodder/extensions/enqueue_in_mediaplayer.py

```
@@ -81,6 +81,122 @@ def open_files(self, filenames):
+                        episodes[0].current_position)
+
+        for episode in episodes:
+            episode.playback_mark()
+            if self.gpodder is not None:
+                self.gpodder.update_episode_list_icons(selected=True)
+
+    def init_dbus(self):
+        bus = gpodder.dbus_session_bus
+
+        if not bus.name_has_owner(self.bus_name):
+            logger.debug('MPRISResumer %s is not there...', self.bus_name)
+            return False
+
+        self.player = bus.get_object(self.bus_name, self.OBJECT_PLAYER)
+        self.signal_match = self.player.connect_to_signal(
+            self.SIGNAL_PROP_CHANGE,
+            self.on_prop_change,
+            dbus_interface=self.INTERFACE_PROPS)
+        return True
+
+    def wait_for_player(self, filename, pos):
+        def name_owner_changed(name, old_owner, new_owner):
+            logger.debug('name_owner_changed "%s" "%s" "%s"',
+                         name, old_owner, new_owner)
+            if name == self.bus_name:
+                logger.debug('MPRISResumer player %s is there', name)
+                cancel.remove()
+                util.idle_add(lambda: self.do_enqueue(filename, pos))
+
+        logger.debug('waiting for player %s to enqueue %s',
+                     self.bus_name, filename)
+        bus = gpodder.dbus_session_bus
+        obj = bus.get_object(self.NAME_DBUS, self.OBJECT_DBUS)
+        cancel = obj.connect_to_signal(
+            'NameOwnerChanged',
+            name_owner_changed,
+            dbus_interface=self.NAME_DBUS)
+
+    def do_enqueue(self, filename, pos):
+        def on_reply():
+            logger.debug('MPRISResumer opened %s', self.url)
+
+        def on_error(exception):
+            logger.error('MPRISResumer error %s', repr(exception))
+            self.signal_match.remove()
+
+        if filename.startswith('/'):
+            try:
+                import pathlib
+                self.url = pathlib.Path(filename).as_uri()
+            except:
```

Is the `except` for `ImportError` (Python 2 and no `pathlib`)? If so, please make it explicit, so that we don't unnecessarily hide other exceptions.